mars/tensor/expressions/linalg/core.py

```diff
@@ -21,7 +21,7 @@
 from ..core import TensorOperandMixin
 
 
-class TSQR(TensorOperandMixin):
+class QRBase(TensorOperandMixin):
```

Comment:
It is better to create separate classes for different QR methods and dispatch in ``qr(a, method='method')``.

Reply:
`TensorQR` is a subclass of `QRBase` now, if create separate classes, each class may provide a staticmethod `tile` ?

Reply:
`tile` is a classmethod, this is not a issue